app/misc/router.ts

```diff
@@ -58,16 +58,29 @@ function checkSignedIn() {
   }
 }
 
+function credentialsEntered() {
+    // for hamburger menu
+    if (continuedWithoutSignIn) {
+        // not logged in, hide 'sign out' button
+        console.log("user is not logged in");
+        document.getElementById("hamsignout").style.display = "none";
+    } else {
+        // logged in, hide 'sign in' button
+        console.log("user is logged in");
+        document.getElementById("hamsignin").style.display = "none";
```

Comment:
I think you could clean this up a touch by factoring out the `document.getElementById...` lines since they are doing the same thing.  The only difference is the variable you are getting, in which case I suggest doing something like this:
```
function credentialsEntered() {
    let status;
    // for hamburger menu
    if (continuedWithoutSignIn) {
        // not logged in, hide 'sign out' button
        console.log("user is not logged in");
        status = "hamsignout";
    } else {
        // logged in, hide 'sign in' button
        console.log("user is logged in");
        status = "hamsignin";
     }
    document.getElementById(status).style.display = "none";
}
```

Reply:
Very nice. I like it.